generators/generator-bot-assistant-core/README.md

```diff
@@ -1,41 +1,94 @@
 # @microsoft/generator-bot-assistant-core [![NPM version][npm-image]][npm-url]
-> Yeoman generator for creating an Adaptive bot built on the Azure Bot Framework using the Assistant Core template.
 
-## Installation
+> Yeoman generator for creating an assistant-style bot with NLP built on the Azure Bot Framework & Composer.
```

Comment:
I went back and forth on this one a lot. I was hesitant to use something referencing the adaptive stack/dialogs here as lots of people would have no idea what that is, but I also wanted to somehow distinguish these from our other Yeoman for generating SDK-first bots. I think I'm ok with dropping Composer, as by the end of the year this will really be the only generator

Reply:
Maybe you can use the declarative label rather than adaptive or composer?

Reply:
Just removed it completely - I don't think it is really information that they need at that point.